ruby/initialize.js

```diff
@@ -177,9 +187,22 @@
         getViewportSize() {
             return addCommand(ruby`@eyes.get_viewport_size`)
         },
-        locate() {
-            return addCommand(ruby`raise 'Eyes locate method havent been implemented'`)
+        locate(visualLocator) {
+            return addCommand(ruby`@eyes.locate({locator_names: ${visualLocator.locatorNames}})`)
         },
+        extractText(regions) {
+            return addCommand(ruby`@eyes.extract_text(${regions})`).type({
+                type: 'Array'
+            })
+        },
+        extractTextRegions(settings) {
+            return addCommand(ruby`@eyes.extract_text_regions(${settings})`).type({
+                type: 'Hash',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
                type: "Hash",
```